### PR TITLE
Add Transfermarkt.de

### PR DESCRIPTION
**Pull request by DavidLiedke**

### CODE REVIEW

File: src/chrome/content/rules/transfermarkt.de.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+			transfermarkt.de
+-->
+<ruleset name="Transfermarkt.de">
+
+	<target host="www.transfermarkt.de" />
+	<target host="bfd.transfermarkt.de" />
+	<target host="jari.transfermarkt.de" />
+	<target host="lb01.transfermarkt.de" />
+	<target host="wap.transfermarkt.de" />
+	<target host="wwrw.transfermarkt.de" />
```
Comment (J0WI) [inline src/chrome/content/rules/transfermarkt.de.xml]: Wildcard DNS

File: src/chrome/content/rules/transfermarkt.de.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<!--
+		404:
+			transfermarkt.de
```
Comment (J0WI) [inline src/chrome/content/rules/transfermarkt.de.xml]: Can be redirected to `www` (but keep the comment to document the issue).

File: src/chrome/content/rules/transfermarkt.de.xml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+	<target host="transfermarkt.de" />
+	<target host="*.transfermarkt.de" />
+		<test url="http://www.transfermarkt.de/" />
+		<test url="http://www.transfermarkt.de/live/counter" />
+		<test url="http://www.transfermarkt.de/navigation/hauptnavi" />
+	
+	<rule from="^http://transfermarkt\.de/"
+			to="https://www.transfermarkt.de/" />
+			<test url="http://transfermarkt.de/" />
```
Comment (J0WI) [inline src/chrome/content/rules/transfermarkt.de.xml]: This test url is already covered by the target.

File: src/chrome/content/rules/transfermarkt.de.xml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+		404:
+			transfermarkt.de
+-->
+<ruleset name="Transfermarkt.de">
+
+	<target host="transfermarkt.de" />
+	<target host="*.transfermarkt.de" />
+		<test url="http://www.transfermarkt.de/" />
+		<test url="http://www.transfermarkt.de/live/counter" />
+		<test url="http://www.transfermarkt.de/navigation/hauptnavi" />
```
Comment (J0WI) [inline src/chrome/content/rules/transfermarkt.de.xml]: The idea of the 3 test urls requirement for wildcards is to have a bunch of subdomains covered. If `www` is the only subdomain you should avoid the wildcard target.

Comment (J0WI) [reply src/chrome/content/rules/transfermarkt.de.xml]: Please add a hint about the wildcard DNS record in the comments.

File: src/chrome/content/rules/transfermarkt.de.xml (hunk 3 above)
Comment (DavidLiedke) [reply src/chrome/content/rules/transfermarkt.de.xml]: https://transfermarkt.de has no content and no redirect to the www-version.  Only http://transfermarkt.de redirects to the www-version. 

What should i do? Remove the target?

Comment (J0WI) [reply src/chrome/content/rules/transfermarkt.de.xml]: The target and the rule are fine. Just remove the redundant test url.

**Comment by DavidLiedke** [comment]
@J0WI done.

**Comment by J0WI** [comment]
LGTM, Thanks!

**Pull request merged**